common/source/docs/common-matekf765-wse.rst

```diff
@@ -106,6 +106,21 @@ With this option, :ref:`SERIAL7_PROTOCOL<SERIAL7_PROTOCOL>` must be set to "23",
 
 Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
 
+If using the Fport with a Frsky receiver you should consider having it on te version 2.1.0 FCC, the newest 2.1.1 seems to have telemtry issues
```

Comment:
If this is a bug in their firmware, we dont want to make it a permanent comment in our wiki....

Reply:
Okay I'm sorry, I'm new to GitHub so...
The thing is I wanted to make something to help others to get faster on the first setup 

I am using this bord and it took me a lot of time to find this and make it work. Same for the rest the overall documentation for the bord on matek is poor and this bord needs a special setup to make fport work. 
Since it doesnt work for the 2.1.1 version and updates are not quick at frsky idk
On the fport documentation it says for the F 765-Wing. Should I ad there that it's the same for the matek F765-WSE ?

Thanks for your time and hope this can help others Niels.

Reply:
PS: the thing is there are so many topics on discuss ardupilot and something like this could help

Reply:
Niels, aside from the FRSky firmware bug in the lastest ACCESS firmware and having to use the earlier version firmware....did you find that using the existing instructions ,ie use TX pin and the parameters mentioned in this page, did NOT work? if so, we would need to confirm and modify....but I personally have setup this controller with FPort on an FRSky R-XSR ACCst using the instructions in the article several times....

Reply:
Yes the tx doesn't work I first tested the settings described below. I think it's the same as the matek F765-Wing this is why on this site https://ardupilot.org/copter/docs/common-FPort-receivers.html. It says to use the RX pin

Reply:
No I already tested it and it's why I wanted to change the wiki....